platforms/android/tangram/src/main/java/com/mapzen/tangram/HttpCallbackBridge.java

```diff
@@ -0,0 +1,20 @@
+package com.mapzen.tangram;
+
+import java.io.IOException;
+
+/**
+ * {@code HttpCallbackBridge} interface implemented internally in {@link MapController#startUrlRequest(String, long)}
+ * Passes network responses from underlying implementation to be processed internally
+ */
+public interface HttpCallbackBridge {
```

Comment:
Do you think "Bridge" is helpful in this name? I saw your comment about this but I'm not sure that this implementation detail is important to the interface.

We could also consider nesting this interface in the definition of `HttpHandler` since they are so closely related. You'd end up calling it something like `HttpHandler.Callback` which seems reasonable.